test/jdk/tools/jpackage/helpers/jdk/jpackage/test/TKit.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Do you know why 2021 is removed?

Reply:
By accident. Restored and changed to 2022